index.html

```diff
@@ -63,8 +67,331 @@ <h1>I'm Gilbert,<br> Glad to See<br> You!</h1>
           </li>
         </ul>
       </div>
+
+      <!--End of Social media Welcome section-->
     </section>
   </main>
+
+  <!--These are my projects-->
+
+  <section class="Works">
+
+    <div class="Card-Works">
+      <div class="Snapshoot-Portfolio">
+        <img src="./media/Snapshoot Portfolio.png" alt="Website" width="98%">
+      </div>
+
+      <div class="Left-Block">
+        <div class="Primary-Text">
+          <h2> Tonic</h2>
+
+          <ul>
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+
+        <div>
+          <p> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+
+        <div>
+          <ul class="Tags">
+            <li class="Tags-html">
+              <a href="#" class="Tags-content">html</a>
+            </li>
+            <li class="Tags-css">
+              <a href="#" class="Tags-content">css</a>
+            </li>
+            <li class="Tags-js">
+              <a href="#" class="Tags-content">javascript</a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="Action">
+          <div class="Normal-button">
+            See Projects
+          </div>
+        </div>
+      </div>
+    </div>
+
+
+    <div class="Card-Works">
+      <div class="Snapshoot-Portfolio">
+        <img src="./media/multi-post.png" alt="Website">
+      </div>
+
+      <div class="Left-Block">
+        <div class="Primary-Text">
+          <h2> Multi-Post Stories</h2>
+
+          <ul>
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+        </div>
+
+        <div>
+          <p> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+
+        <div>
+          <ul class="Tags">
+            <li class="Tags-html">
+              <a href="#" class="Tags-content">html</a>
+            </li>
+            <li class="Tags-css">
+              <a href="#" class="Tags-content">css</a>
+            </li>
+            <li class="Tags-js">
+              <a href="#" class="Tags-content">javascript</a>
+            </li>
+          </ul>
+        </div>
+
+        <div class="Action">
+          <div class="Normal-button">
+            See Projects
+          </div>
```

Comment:
- I think it will be better if your "see projects" is in a `<button>` tag because this is used to define a clickable button instead of a `<div>` 😄 

- Great job implementing the `hover` effect on all the buttons of your project 🍷